src/components/DefineCompany/OfficeAddresses.vue

```diff
@@ -0,0 +1,518 @@
+<template>
+  <div id="office-addresses">
+    <div v-if="!isEditing">
+      <v-layout row>
+        <v-flex md4><label><strong>Registered Office</strong></label></v-flex>
+        <v-flex md4>
+          <label><strong>Mailing Address</strong></label>
+          <mailing-address
+            :address="mailingAddress"
+            :editing="isEditing"
```

Comment:
I think this prop defaults to false, and this is the "read only" markup anyway, so I suggest either removing this prop or explicitly setting it to false (so it's obvious that this value doesn't change here).

And ditto for other base address components below.